plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergAnalyzeProperties.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.iceberg;
+
+import com.google.common.collect.ImmutableList;
+import io.trino.spi.TrinoException;
+import io.trino.spi.session.PropertyMetadata;
+import io.trino.spi.type.ArrayType;
+
+import javax.inject.Inject;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.Set;
+
+import static com.google.common.collect.ImmutableSet.toImmutableSet;
+import static io.trino.spi.StandardErrorCode.INVALID_ANALYZE_PROPERTY;
+import static io.trino.spi.type.VarcharType.VARCHAR;
+import static java.lang.String.format;
+
+public class IcebergAnalyzeProperties
+{
+    public static final String COLUMNS_PROPERTY = "columns";
+
+    private final List<PropertyMetadata<?>> analyzeProperties;
+
+    @Inject
+    public IcebergAnalyzeProperties()
+    {
+        analyzeProperties = ImmutableList.<PropertyMetadata<?>>builder()
+                .add(new PropertyMetadata<>(
```

Comment:
There's a copy of this in three places now. Is it worth having a `CommonAnalyzeProperties` in `trino-plugin-toolkit`?

Reply:
no, i don't think so